src/library_glemu.js

```diff
@@ -185,9 +185,9 @@ var LibraryGLEmulation = {
 
 
       // Add some emulation workarounds
-      err('WARNING: using emscripten GL emulation. This is a collection of limited workarounds, do not expect it to work.');
+      console.warning('WARNING: using emscripten GL emulation. This is a collection of limited workarounds, do not expect it to work.');
 #if GL_UNSAFE_OPTS == 1
-      err('WARNING: using emscripten GL emulation unsafe opts. If weirdness happens, try -sGL_UNSAFE_OPTS=0');
+      console.warning('WARNING: using emscripten GL emulation unsafe opts. If weirdness happens, try -sGL_UNSAFE_OPTS=0');
```

Comment:
Do you see these more than once in you program?  We could use the warnOnce helper here maybe?

Reply:
There are at least 10 such warnings. I'll use the warnOnce.

Reply:
Hmm, this message shows up for each JS wroker, so I don't think that warnOnce will help here?

Reply:
Hmm.. why is GL being initialized on each worker?   Is that intended?

Reply:
We use the predefined thread pool (`-sPTHREAD_POOL_SIZE='navigator.hardwareConcurrency`) in order to synchronously spawn threads within our code. I suspect that worker initialization automatically triggers GL init method?

Reply:
Indeed that seems to be the case.   I wonder if we can/should avoid initializing GL on threads that are not using GL (i.e. can we call init() lazily?)